src/microbe_stage/Microbe.Contact.cs

```diff
@@ -610,14 +610,19 @@ public void SendOrganellePositionsToMembrane()
             throw new InvalidOperationException("Microbe must be initialized first");
 
         var organellePositions = new List<Vector2>();
+        bool hasMultihexOrganelles = false;
```

Comment:
Is this a workaround for https://github.com/Revolutionary-Games/Thrive/issues/4117 ? If so could you add a TODO comment here linking to that issue?

Reply:
If that is the case the issue should also be linked from the documentation on `HasMultihexOrganelles`

Reply:
Yeah, it's a workaround. With the comment, do you mean something like `// TODO: This is a workaround for https://github.com/Revolutionary-Games/Thrive/issues/4117. Check if this can be removed once that issue is fixed.`?

Reply:
Yeah, that's about fine wording for the TODO comment. (though maybe remove the `.` after the link in case it makes the link unclickable for someone as some editors might consider the dot to be part of the link incorrectly).